src/streaming/utils/Capabilities.js

```diff
@@ -187,13 +188,16 @@ function Capabilities() {
 
     /**
      * Check if a specific EssentialProperty is supported
-     * @param {object} ep
+     * @param {DescriptorType} ep
      * @return {boolean}
      */
     function supportsEssentialProperty(ep) {
         let supportedEssentialProps = settings.get().streaming.capabilities.supportedEssentialProperties;
+
         try {
-            return supportedEssentialProps.indexOf(ep.schemeIdUri) !== -1;
+            let descriptor = new DescriptorType();
+            descriptor.init(ep);
+            return descriptor.inArray(supportedEssentialProps);
```

Comment:
Will check tomorrow... sounds like I need one more unit test as well.

Reply:
Sorry, I can't reproduce that.
Playing `http://localhost:3000/samples/thumbnails/thumbnails.html` gives me:
![image](https://github.com/Dash-Industry-Forum/dash.js/assets/92983372/0f49aa4f-323f-4e00-879a-5b37c81abb8c)
when using these changes:
![image](https://github.com/Dash-Industry-Forum/dash.js/assets/92983372/e1b6d0fb-6d39-4265-a2cd-94246a449165)

I also replaced the "multiple_tiles"-MPD from the example with the link you provided - same result.

Furthermore, this case is also tested in unit-tests: 
https://github.com/DolbyLaboratories/dash.js/blob/fee2223b7557e03d4f0a26db2bd1b6a09f868e1e/test/unit/streaming.utils.Capabilities.js#L72
Ok, it used `value: 'myvalue'` instead of  `value="10x1"`, but that does not make any difference.